dev/io.openliberty.reporting.internal/src/io/openliberty/reporting/internal/ReporterTask.java

```diff
@@ -56,7 +60,9 @@ public void run() {
         try {
             DataCollector collector = new DataCollector(featureProvisioner, fixManager, serverInfo);
             Map<String, String> data = collector.getData();
-            String urlLink = (String) props.get("urlLink");
+            Map<String, ? extends ProductInfo> allProductInfo = ProductInfo.getAllProductInfo();
```

Comment:
To save two calls to getAllProductInfo (here and another in Datacollector), i would call this up and this level and pass it through to the DataCollector like we do with the other objects, as this will save time parsing all the files and means we only have to handle parsing issues here and not in two places

Reply:
Done